### PR TITLE
Rha/noetic devel rickstaa add franka gazebo simulation

### PR DESCRIPTION
**Pull request by rickstaa**
This pull request adds the [franka_gazexbo](https://github.com/frankaemika/franka_ros/tree/develop/franka_gazebo) gazebo simulation to the panda_moveit_config package. To keep the Noetic branch as close as possible to the melodic-branch I also used the `position_joint_trajectory_controller`. For this controlller to work https://github.com/frankaemika/franka_ros/pull/181 has to be merged in the upstream.

### Remarks

The pid gains need to be improved in the upstream pull request. I think we however can already merge #8.

### CODE REVIEW

**Review by rhaschke — APPROVED** [review]
Looks good to me. Is this ready to merge?

**Comment by rickstaa** [comment]
@rhaschke I think from our side it is ready. The only thing that has to be done is improve the PID gains found in https://github.com/frankaemika/franka_ros/pull/181 and merge that pull request.

File: config/panda_arm.xacro [hunk 1]
```diff
@@ -7,7 +7,6 @@
         <!--CHAINS: When a chain is specified, all the links along the chain (including endpoints) are included in the group. Additionally, all the joints that are parents to included links are also included. This means that joints along the chain and the parent joint of the base link are included in the group-->
         <!--SUBGROUPS: Groups can also be formed by referencing to already defined group names-->
         <group name="panda_arm">
-            <joint name="virtual_joint"/>
```
Comment (rhaschke) [inline config/panda_arm.xacro:10]: Obviously, for Gazebo it's important to not list the virtual joint here. This should be fixed in MSA as well. Could you please add a TODO?

**Review by rhaschke — APPROVED** [review]

File: config/panda_arm.xacro (hunk 1 above)
Comment (rickstaa) [reply config/panda_arm.xacro:10]: @rhaschke Done.

**Pull request merged**